### DISCUSSION TITLE
"Billable page views" vs "Total page views"

### DISCUSSION THREAD

**Discussion by chulinguy** [plausible/analytics#2385] [Anything Else]
Dear Plausible Staff, 

Can someone explain the difference between "Billable page views" and "Total page views" ?
On our site, billable page views is about twice the number of total page views.  

Thank you in advance!

**Comment by metmarkosaric** [comment]
Thanks for reaching out @chulinguy!

Billable pageviews include both the regular pageviews but also any goals and custom events that you may be sending our way. 

If you go into your account settings, there's the "Your usage" section which lists the the total number of billable pageviews in last 30 days and there you also have a split between pageviews and custom events.

I hope this helps, thanks!
